src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
The reason I suggested it is that there isn't enough visual separation between the two sentences. My preferred solution would be to have a blank line, but that would require more blank lines elsewhere and I was trying to conserve vspace. :-(

I'm OK with keeping as is.

But I still vote for s/:// here.